wiki/Tournaments/oPHN/2023_taiko/en.md

```diff
@@ -0,0 +1,337 @@
+3. Matches will use ScoreV2 and NoFail in Head-to-Head mode.
+4. Players and staff must be treated with respect. Instructions from the hosts and referees are to be followed. Abusing the rules and poor sportsmanship may result in disqualification from the entire tournament.
+5. Any modification from the ruleset will be announced. Players must be active on the tournament's discord server in the case of these modifications.
+
+### Mappool structure
+
+- There will be separate mappools for each stage. Each mappool contains the following mod brackets: NoMod, Hidden, HardRock, DoubleTime, ForceMod, and Tiebreaker. There will be no Tiebreaker for Qualifiers.
+  - DoubleTime can be played as Nightcore if all players in the lobby agree to it.
+  - ForceMod will be played either with Hidden, HardRock, or Hidden + HardRock.
+  - Tiebreaker will be played with NoMod and will be only played in case of a tie.
+- The win condition and mappool size per stage are as follows:
+
+| Round stage | Best of | NM | HD | HR | DT | FM | TB |
+| :-- | :-: | :-: | :-: | :-: | :-: | :-: | :-: |
+| Qualifiers | - | 3 | 1 | 1 | 1 | 1 | - |
+| Round of 16 | 9 | 4 | 2 | 2 | 2 | 3 | 1 |
+| Quarterfinals | 11 | 5 | 2 | 2 | 2 | 3 | 1 |
+| Semifinals | 11 | 5 | 2 | 2 | 2 | 3 | 1 |
+| Finals | 13 | 6 | 2 | 2 | 2 | 3 | 1 |
+| Grand finals | 13 | 6 | 2 | 2 | 2 | 3 | 1 |
+
+### Qualifiers regulations
+
+1. Players must sign up to one of the qualifier lobbies provided by the hosts. Instructions will be announced after the Qualifiers mappool showcase. Players may request a special lobby if needed, as long as a referee is available at the requested time. A player that doesn’t pick a time slot before Thursday 23:59 will be disqualified.
+2. There will be no warmups for all qualifiers lobbies. The mappool will be played twice during a lobby, and will be played in the following order: NM1, NM2, NM3, HD1, HR1 ,DT1, FM1. A 5-minute break will be offered between the first and second run of the mappool, which can be skipped if all players in the lobby are ready. A player may skip certain maps during their second run if they so desire.
+3. The player’s highest score for each map will be used for seeding. Players will be ordered by overall rank, which is computed by taking the average of their rankings on each map. In case of a tie, the sum of the top scores of the tied players are taken into account instead. Sharing of scores before the results showcase is discouraged.
+4. The number of players that will advance to the next stage will depend on the total number of participants.
+
+### Match regulations
+
+1. Each player is allowed (but not required) to have a warm-up with length below 3:00. Players may request to use DoubleTime if the map length is slightly above the restriction. The players must give the map link to their referee 10 minutes before their match. Otherwise, the player’s warm-up is skipped.
```

Comment:
```suggestion
1. Each player is allowed (but not required) to have a warm-up with length below 3:00. Players may request to use Double Time if the map length is slightly above the restriction. The players must give the map link to their referee 10 minutes before their match. Otherwise, the player’s warm-up is skipped.
```